components.yml

```diff
@@ -9,19 +9,24 @@ deployed_component_versions:
   # component-identifier: component-version
```

Comment:
There should not be any changes in this file as part of the PR. Did you base your branch off of integration instead of master?

Reply:
Oops. Let me base this PR of again off integration. Yes, I realised I originally branched off integration for these changes.